packages/cli/src/fingerprint/globber.ts

```diff
@@ -48,13 +62,27 @@ class Globber extends EventEmitter {
       if (!files.has(f)) this.remove(f);
     }
 
-    this.emit('end');
+    this.updateStats(found, 0, () => {
+      this.emit('end');
+
+      if (this.running) this.timeout = setTimeout(this.scanNow.bind(this), this.interval);
```

Comment:
This timeout should probably be `unref()`-ed to make it behave nicer on termination.

Reply:
Ah, sorry, my mistake, it can't be unref-ed — if you unref it then if the watcher is down the process would exit immediately after one loop. Same goes for the stat timeout.